install_config/web_console_customization.adoc

```diff
@@ -334,3 +324,47 @@ Header] and OAuth or
 link:../install_config/configuring_authentication.html#OpenID[OpenID] identity
 providers, which require visiting an external URL to destroy single sign-on
 sessions.
+
+[[web-console-customization-additional-examples]]
+== Additional Examples
+
+For more examples showing common ways you can customize the web console,
+additional extension examples are available in the
+https://github.com/openshift/origin-web-console/tree/master/extensions/examples[*_/extensions/examples/_*]
+directory of the upstream
+https://github.com/openshift/origin-web-console[OpenShift Management Console]
+source repository on GitHub.
+
+The JS file in the directory shows how to add to one of the extension points. In
+addition, several other examples are provided in the
+https://github.com/openshift/origin-web-console/tree/master/app/scripts/extensions[*_/app/scripts/extensions/_*]
+directory. They adhere to the following pattern:
```

Comment:
I like "adhere to" (and quick grep shows one other instance in `creating_images/custom.adoc`), but maybe "take" would work, too.